src/components/squareComponent/square.spec.js

```diff
@@ -14,4 +14,55 @@ describe('Square Component', () => {
     ReactDOM.unmountComponentAtNode(gameSpaceDiv);
   });
 
+  it('tests the square is empty before the user has placed a marker on the square', () => {
+    const wrapper = mount(<GameSquare />);
+    expect(wrapper.state('valueX')).toBe('');
+    expect(wrapper.state('valueO')).toBe('');
+  });
+
+  it('handles placing an X on the square', () => {
+    const wrapper = mount(<GameSquare />);
+    const instance = wrapper.instance();
+    expect(wrapper.state('valueX')).toBe('');
+    instance.toggleX();
+    wrapper.find('.game-spaces').simulate('click');
+    expect(wrapper.state('valueX')).toBe('X');
+  });
+
+  it('handles placing an O on the square', () => {
```

Comment:
I would group these unit tests into two separate `describe` calls, one highlighting each "action."

Maybe something like this. Note, this example is neither complete, not likely 100% accurate:
```javascript
describe('Square Component', () => {
  describe('Mounting the component', () => {
    it('is empty', () => {
    });
  });
  describe('Clicking a square', () => {
    it('places an X', () => {
    });
    it('does a different thing', () => {
    });
  });
});
```

Reply:
Unit tests are now grouped into separate `describe calls`